sys-auth/fprintd/fprintd-1.94.2.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit meson pam python-any-r1 systemd
+
+MY_P="${PN}-v${PV}"
+
+DESCRIPTION="D-Bus service to access fingerprint readers"
+HOMEPAGE="https://gitlab.freedesktop.org/libfprint/fprintd"
+SRC_URI="https://gitlab.freedesktop.org/libfprint/${PN}/-/archive/v${PV}/${MY_P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm ~arm64 ~ia64 ppc ~ppc64 ~riscv sparc x86"
+IUSE="doc pam systemd test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/dbus-glib
+	dev-libs/glib:2
+	>=sys-auth/libfprint-1.94.0:2
```

Comment:
Two possible changes:
- I don't know if they always release these in sync (libfprint & fprintd), but if they do, >=sys-auth/libfprint-${PV}:2
- If they don't (I would do this if you're not sure, as it's possible they will at some point), >=sys-auth/libfprint-$(ver_cut 1-2):2

This acts as a nice guard to ensure someone doesn't bump one without the other and the dep stays fresh.

Ignore this if sometimes the versions are totally out of sync/not released at same time.

Reply:
They used to be synced but I've noticed they aren't since 1.90.x

Reply:
OK, fair enough